src/app/content/search/components/SearchResultsSidebar/index.spec.tsx

```diff
@@ -175,4 +184,24 @@ describe('SearchResultsSidebar', () => {
 
     expect(dispatch).toHaveBeenCalledWith(clearSearch());
   });
+
+  it('fixes overscroll in safari', () => {
+    const {tree} = renderToDom(render());
+    const fixForSafariMock = jest.spyOn(domUtils, 'fixSafariScrolling');
+
+    store.dispatch(requestSearch('cool search'));
+    store.dispatch(receiveSearchResults(makeSearchResults()));
+
+    const sidebar = ReactTestUtils.findRenderedComponentWithType(tree, SearchResultsBarWrapper);
+
+    jest.useFakeTimers();
+    setTimeout(() => {
```

Comment:
i don't think you need this `setTimeout`

Reply:
I tried removing it. Didn't work, don't really know what you mean.